src/Accounts/Accounts/help/Connect-AzAccount.md

```diff
@@ -545,6 +545,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -MicrosoftGraphAccessToken
+AccessToken for Microsoft Graph Service
```

Comment:
```suggestion
Access token to Microsoft Graph
```

Reply:
In addition, please change description of `GraphAccessToken` to `Access token to Azure AD Graph API`